examples/covariance/plot_outlier_detection.py

```diff
@@ -62,7 +68,7 @@
     X = np.r_[X, np.random.uniform(low=-6, high=6, size=(n_outliers, 2))]
 
     # Fit the model with the One-Class SVM
-    plt.figure(figsize=(10, 5))
+    plt.figure(figsize=(15, 5))
```

Comment:
did you check that it does render fine in the doc? I suspect it will be too large